elements/rh-skeleton/rh-skeleton-lightdom.css

```diff
@@ -0,0 +1,122 @@
+rh-skeleton {
+  background: light-dark(rgba(0, 0, 0, 0.1), rgba(255, 255, 255, 0.1));
+  block-size: var(--rh-length-lg, 16px);
+  border-radius: var(--rh-border-radius-default, 3px);
+  display: block;
+  inline-size: 100%;
+  margin-block-end: var(--rh-space-xs, 4px);
+  overflow: hidden;
+  position: relative;
+
+  &:after {
+    --_placeholder-color:
+      light-dark(var(--rh-color-black, #000000),
+        var(--rh-color-white, #ffffff));
+
+    animation: shimmer 2s infinite;
+    background:
+      linear-gradient(to right,
+        hsla(from var(--_placeholder-color) h s l / 0) var(--rh-opacity-0, 0%),
+        hsla(from var(--_placeholder-color) h s l / 0.1) calc(var(--rh-opacity-20, 20%) + 5%),
+        hsla(from var(--_placeholder-color) h s l / 0.25) var(--rh-opacity-50, 50%),
+        hsla(from var(--_placeholder-color) h s l / 0.1) calc(var(--rh-opacity-70, 70%) + 5%),
+        hsla(from var(--_placeholder-color) h s l / 0) var(--rh-opacity-100, 100%));
+    content: '';
+    inset: 0;
+    position: absolute;
+    transform: translateX(-100%);
+  }
+}
+
+@keyframes shimmer {
+  100% {
+    transform: translateX(100%);
+  }
+}
+
+rh-skeleton[size='xs'] {
+  block-size: calc(var(--rh-length-sm, 6px) * 3);
```

Comment:
xs => sm?
sm => xl - 3?
md => xl?
lg => xl + 3?


this is confusing, no? Do we even need a size attribute? can't users use css for presentation details like this?